pkg/operator/controller/controller_dns_daemonset_test.go

```diff
@@ -214,6 +214,14 @@ func TestDaemonsetConfigChanged(t *testing.T) {
 			},
 			expect: true,
 		},
+		{
+			description: "if the termination grace period is defaulted",
+			mutate: func(daemonset *appsv1.DaemonSet) {
+				thirty := int64(30)
```

Comment:
Can we get the API defaulted value right from the `corev1` API? 
(Not that I anticipate the value changing anytime soon, but IMO using the API constant would make the test cleaner)

https://github.com/kubernetes/api/blob/master/core/v1/types.go#L2601

Reply:
Nifty!  Thanks for the pointer!  Done.